README.md

```diff
@@ -2,127 +2,84 @@
-
-```json
-{
-  "extensions": {
-    "<page_path>/myshelf": {
-      "component": "vtex.shelf/Shelf"
-    },
+  dependencies: {
+    "vtex.shelf": "1.x"
   }
-}
-```
-
-> 3- On your react component that contains the Shelf
-```javascript 
-import { ExtensionPoint } from 'vtex.render-runtime'
-...
-render() {
-  return (
-    <ExtensionPoint id="myshelf" />
-  )
-}
-...
-```
-
-> 4- To run your Shelf App you should run on your workspace the command:
-
-```sh
-$ vtex link
 ```
 
+Then, add `shelf` block into our app theme, like we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
 
-## Schema Properties (Used By Editor)
-
-``` javascript
-- category // Category ID of the listed items in the shelf
-  - Type: Number
-- collection // Collection ID of the listed items in the shelf.
-  - Type: Number
-- orderBy // Ordenation type of the items in the shelf.
-  - Type: String
-  - Default: 'OrderByTopSaleDESC'
-  - Enum: ['OrderByTopSaleDESC', 'OrderByPriceDESC', 'OrderByPriceASC']
-- maxItems // Maximum number of items in the shelf.
-  - Type: Number
-  - Default: 10
-- itemsPerPage // Maximum number of items on the page.
-  - Type: Number
-  - Default: 5
-- scroll // Scroll type of slide transiction.
-  - Type: String
-  - Default: 'BY_PAGE'
-  - Enum: ['BY_PAGE', 'ONE_BY_ONE']
-- arrows // If the arrows are showable or not.
-  - Type: Boolean
-  - Default: true
-- titleText // Title of the shelf.
-  - Type: String
-  - Default: ''
-- summary // Product Summary schema properties
-  - Type: Object
-  - Properties
-    - showListPrice
-      -type: Boolean
-      -Default: true
-    - showLabels // Show product's prices' labels
-      -type: Boolean
-      -Default: true
-    - showInstallments // Show product's payment installments
-      -type: Boolean
-      -Default: true
-    - showBadge // Show the discount badge
-      -type: Boolean
-      -Default: true
-    - badgeText // Badge's text
-      -type: String
-    - buyButtonText // Custom buy button's text
-      -type: String
-    - displayBuyButton // Buy button display types.
-      - Type: String
-      - Default: 'DISPLAY_ALWAYS'
-      - Enum: ['DISPLAY_ALWAYS', 'DISPLAY_ON_HOVER', 'DISPLAY_NONE']
-```
+### Blocks API
+:construction: :construction: :construction:
 
-## CSS Classes
-
-```css
-/* ==== '/react/CustomClasses.js' ==== */
-vtex-shelf
-vtex-shelf__title-text
-vtex-shelf__title-content
-vtex-shelf__slide
-```
+This app have an interface that describes what are the rules that the blocks must implement when you want to use the shelf block. 
 
-## Tests
-
-Run the tests with the command
-```
-cd react && yarn test
+```json
+  "shelf": {
+    "component": "Shelf"
+  },
+  "related-products": {
+    "component": "RelatedProducts"
+  }
 ```
+#### Configuration 
+Through the Storefront you can change the behavior and interface of shelf. But, you also can make adjusts in your theme app, like Dreamstore does.
```

Comment:
`But, you can also make adjustments...`

Reply:
`Through the Storefront, you can change the shelf's behavior and interface. However, you also can make in your theme app, as Dreamstore does.`